protocol/communities/manager.go

```diff
@@ -3224,6 +3224,26 @@ func (m *Manager) GetByIDString(idString string) (*Community, error) {
 	return m.GetByID(id)
 }
 
+func (m *Manager) GetCommunityShard(communityID string) (*shard.Shard, error) {
```

Comment:
Can `GetCommunityShard` and `SaveCommunityShard` accept a communityId in the same format, such as `types.HexBytes`?

Reply:
fixed